beacon-chain/rpc/prysm/v1alpha1/beacon/van_pending_blocks.go

```diff
@@ -172,7 +173,13 @@ func (bs *Server) StreamNewPendingBlocks(
 					return status.Errorf(codes.Internal, "Could not send over stream: %v", err)
 				}
 
-				blockInfo, err := prepareBlockInfo(unwrappedBlk)
+				// orchestrator needs to know about syncing status
+				var disableDelete bool
+				if !bs.PendingQueueFetcher.OrcVerification() {
```

Comment:
Should we introduce `EnableVanguardNode` check here? (I mean in gRPC Beacon Chain service server struct first). Or by design it means that vanguard node is enabled and we don't need to worry about this? (Because it's `van_pending_blocks`). Motivation would be to stick with `beacon-chain/blockchain/process_block.go` implementation for example.